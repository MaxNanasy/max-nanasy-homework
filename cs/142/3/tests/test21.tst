type newType = short;
var newType : newType;
const newType = 10;
   void newType (int a, short b, newType c) {
   PRINT SYMBOL TABLE
   }
