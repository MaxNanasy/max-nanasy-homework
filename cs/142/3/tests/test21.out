Error: SEMANTIC: REDEFINED SYMBOL
***** Symbol Table Contents *****
N: int C: TYPE T: null
N: void C: TYPE T: null
N: short C: TYPE T: null
N: newType C: TYPE T: short
N: newType C: VAR T: newType
